test/run

```diff
@@ -3,6 +3,15 @@
 # This is the expected entry point for Cockpit CI; will be called without
 # arguments but with an appropriate $TEST_OS, and optionally $TEST_SCENARIO
 
+# Currently supported scenarios:
+# devel       - runs tests with coverage enabled and generates a html file with coverage information
+# pybridge    - runs tests with the Python bridge (soon to be deprecated after all images are moved over to the Python bridge)
```

Comment:
No, this is, and remains relevant exactly for RHEL/CentOS 8. This won't ever move to the Python bridge.